concourse/pipelines/destroy-deployer.yml

```diff
@@ -78,7 +87,7 @@ jobs:
             [ "$(cat concourse-bosh-state/concourse-manifest-state.json)" = "{}" ] \
               && echo Concourse bosh state empty, assuming already destroyed \
               && exit 0
-            echo -n "${private_ssh_key}" > id_rsa
```

Comment:
It was removed in https://github.com/alphagov/paas-cf/commit/4e4c5fce2c85e42226aa801ebd6df54469dbe565 but destroy-deployer wasn't completely fixed at the time.